src/System.Management.Automation/engine/Utils.cs

```diff
@@ -895,113 +895,61 @@ internal static bool IsAdministrator()
 #endif
         }
 
-        internal static bool NativeItemExists(string path)
+        internal static bool ItemExists(string path)
         {
-            bool unusedIsDirectory;
-            Exception unusedException;
+            bool result = false;
+            try
+            {
+                result = ItemExists(path, out bool unused);
```

Comment:
> ItemExists(path, out bool unused)

I suggest changing it to `ItemExists(path, out _)`, using the "discard" variable.

Reply:
And, maybe

```
internal static bool ItemExists(string path)
{
    try { return ItemExists(path, out _) } catch { }
    return false;
}
```